src/lavinmq/http/controller/vhost-limits.cr

```diff
@@ -0,0 +1,89 @@
+require "../controller"
+require "../../sortable_json"
+
+module LavinMQ
+  module HTTP
+    class VHostLimitsController < Controller
+      private def register_routes
+        get "/api/vhost-limits" do |context, _params|
+          vhosts = vhosts(user(context)).compact_map { |v| VHostLimitsView.new(v).self_if_limited }
+          page(context, vhosts)
+        end
+
+        get "/api/vhost-limits/:vhost" do |context, params|
+          with_vhost(context, params) do |vhost|
+            refuse_unless_management(context, user(context), vhost)
+            v = @amqp_server.vhosts[vhost]
+            [VHostLimitsView.new(v).self_if_limited].compact.to_json(context.response)
+          end
+        end
+
+        put "/api/vhost-limits/:vhost/:type" do |context, params|
+          u = user(context)
+          refuse_unless_administrator(context, u)
+          context.response.status_code = 400
+          with_vhost(context, params) do |vhost|
+            v = @amqp_server.vhosts[vhost]
+            if body = context.request.body
+              json = JSON.parse(body)
+              if value = json["value"]?.try &.as_i?
+                value = nil if value < 0
```

Comment:
Should we reject the request if it's "invalid", now we silently remove the limit.

Reply:
Only 204 on valid values, 400 for all other requests

Reply:
Yeah, I was thinking we make `-1` invalid. Now it's valid, but setting nil.

Reply:
It's correct behavior, it's the way to delete the limit

Reply:
Same as for rabbitmq